templates/telco/index.html

```diff
@@ -0,0 +1,242 @@
+{% extends '_layout.html' %}
+
+
+{% block title %}Telco. The 5G mobile subscriber experience.{% endblock %}
+{% block meta_description %}Delighting network customers with 5G apps.
+{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1DEG0SELYXIJCdqwaJOWM6R1yCiTnEub4kaK0aX4bT2I/edit#heading=h.44dtjnxyfix2#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<div class="wrapper">
```

Comment:
Not sure we need these divs line 11 and 12? The <section> should be enough as we have the `{% block content %}`